apps/cli/src/index.ts

```diff
@@ -1,10 +1,24 @@
 import { Command } from "commander";
 
+import { makeConsoleLogger } from "./adapters/console/index.js";
+import { makeRepositoryReader } from "./adapters/node/index.js";
+import { makeNodeReader } from "./adapters/yarn/index.js";
+import { doctor } from "./commands/index.js";
+import { Dependencies } from "./domain/dependencies.js";
+
+const deps: Dependencies = {
+  logger: makeConsoleLogger(),
+  nodeReader: makeNodeReader(),
+  repositoryReader: makeRepositoryReader(),
+};
+
 const program = new Command();
 
 program
   .name("DX-CLI")
```

Comment:
the name should be aligned by the `bin` script 
```suggestion
  .name("dx")
```